gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -1086,3 +1033,90 @@ verboseHandlerHeapResize(J9HookInterface** hook, uintptr_t eventNum, void* event
 {
 	((MM_VerboseHandlerOutput*)userData)->handleHeapResize(hook, eventNum, eventData);
 }
+
+void
+MM_VerboseHandlerOutput::outputInitializedStanza(MM_EnvironmentBase *env, MM_VerboseManager *manager, MM_VerboseBuffer *buffer)
+{
+
+	MM_GCExtensionsBase* extensions = env->getExtensions();
+	
+	
+	UDATA numaNodes = extensions->_numaManager.getAffinityLeaderCount();
+	char tagTemplate[200];	
+	
+	OMRPORT_ACCESS_FROM_OMRPORT(env->getPortLibrary());
+	manager->setInitializedTime(omrtime_hires_clock());	
+	manager->getVerboseHandlerOutput()->getTagTemplate(tagTemplate, sizeof(tagTemplate), manager->getIdAndIncrement(), omrtime_current_time_millis());
```

Comment:
Going to manager and getting handler is redundant, `getTagTemplate` can be called directly here